modules/apps/frontend-js/frontend-js-react-web/npmscripts.config.js

```diff
@@ -14,20 +14,17 @@
 
 module.exports = {
 	build: {
-		bundler: {
-			exclude: {
-				lodash: false,
-			},
-			ignore: ['**/legacy/config.js'],
-		},
-		exports: {
-			'classnames': 'classnames/index',
-			'formik': 'formik/dist/index',
-			'prop-types': 'prop-types/index',
-			'react': 'react/index',
-			'react-dnd': 'react-dnd/dist/cjs/index',
-			'react-dnd-html5-backend': 'react-dnd-html5-backend/dist/cjs/index',
-			'react-dom': 'react-dom/index',
-		},
+		bundler: false,
+		tsc: false,
```

Comment:
It's interesting we would need to manually set this, I would think that this would always be false because even if we imported from a ts project, wouldnt we import the build js files?

Reply:
:thinking: These are intermediate flags, but in the end we will only have none. Currently I need to tweak them as I progress in migrating each module.

Regarding tsc, it should always be true, because we want to check the syntax, even if the real works is then done by webpack.